src/obs-ndi-source.cpp

```diff
@@ -573,6 +581,19 @@ void *ndi_source_thread(void *data)
 				break;
 			}
 
+			// Force a clean (blank) frame when settings change to Audio only
+			if (recv_desc.bandwidth ==
+			    NDIlib_recv_bandwidth_audio_only) {
+				obs_source_output_video(obs_source,
+							blank_video_frame());
```

Comment:
This allocates a blank video frame and never releases it

Reply:
agreed!